meta-resin-common/recipes-core/systemd/systemd/coredump.conf

```diff
@@ -0,0 +1,2 @@
+[Coredump]
+Storage=none
```

Comment:
This means that on development we will have the default Storage=external which can easily fill up the state partition. I proposed in the other PR to store the coredump in the journal so we can take advantage of the log rotation there.